cdap-data-fabric/src/test/java/co/cask/cdap/data2/metadata/dataset/BusinessMetadataDatasetTest.java

```diff
@@ -43,7 +44,13 @@
   private static final Id.DatasetInstance datasetInstance =
     Id.DatasetInstance.from(DatasetFrameworkTestUtil.NAMESPACE_ID, "meta");
 
-  BusinessMetadataDataset dataset;
+  private BusinessMetadataDataset dataset;
+
+  private final Id.Application app1 = Id.Application.from("ns1", "app1");
+  // Have to use Id.Program here because Id.Flow does not implement equals. It needs to be updated.
```

Comment:
probably, but I'd refrain from doing it right now. We should report a jira to refactor that entire class - to use consistent methods of object creation, use java.util.Objects for comparison etc.